modules/apps/object/object-api/src/main/java/com/liferay/object/field/setting/util/ObjectFieldSettingUtil.java

```diff
@@ -111,6 +111,20 @@ public static String getValue(String name, ObjectField objectField) {
 		return getValue(name, objectField.getObjectFieldSettings());
 	}
 
+	public static boolean isUnique(
+		List<ObjectFieldSetting> objectFieldSetting) {
+
+		if (objectFieldSetting.isEmpty()) {
```

Comment:
If objectFieldSetting is always != null, I think it would be covered by the next line of code, so this is no necessary